mesonbuild/interpreter.py

```diff
@@ -3658,7 +3658,7 @@ def func_configure_file(self, node, args, kwargs):
         # for backwards compatibility. That was the behaviour before
         # 0.45.0 so preserve it.
         idir = kwargs.get('install_dir', None)
-        if isinstance(idir, str) and idir:
+        if kwargs.get('install', True) and isinstance(idir, str) and idir:
```

Comment:
Wondering if we should warn in the case we have `install:true,install_dir:''`

Reply:
Yeah, probably it would be better.